system-configuration/src/network_configuration.rs

```diff
@@ -0,0 +1,427 @@
+use core_foundation::array::CFArray;
+use core_foundation::base::{CFType, TCFType};
+use core_foundation::base::kCFAllocatorDefault;
+use core_foundation::dictionary::CFDictionary;
+use core_foundation::string::{CFString, CFStringRef};
+
+use dynamic_store::SCDynamicStoreBuilder;
+pub use system_configuration_sys::network_configuration::*;
+use system_configuration_sys::preferences::SCPreferencesCreate;
+
+use std::{fmt, ptr};
+use std::net::IpAddr;
+
+#[derive(Debug)]
+pub struct SCNetworkInterfaceMTU {
+    cur: u32,
+    min: u32,
+    max: u32,
+}
+
+impl SCNetworkInterfaceMTU {
```

Comment:
This struct is used purely as a data struct, it does not carry any logic with it. Then we could simplify things by making all fields in it public and remove the `impl`. That would allow a user to just access the data directly instead of going through getters.